app/helpers/data_getter.py

```diff
@@ -422,21 +422,21 @@ def get_all_published_events(include_private=False):
         return events
 
     @staticmethod
-    def get_call_for_speakers_events(include_private=False):
+    def get_speakers_calls_events(include_private=False):
         events = DataGetter.get_all_published_events(include_private)
         return [event for event in events
-                if DataGetter.get_open_call_for_papers(event.id)
+                if DataGetter.get_open_speakers_calls(event.id)
                 and not event.deleted_at][:12]
 
     @staticmethod
-    def get_open_call_for_papers(event_id):
+    def get_open_speakers_calls(event_id):
         datetime_now = datetime.datetime.now()
-        return CallForPaper.query.filter_by(
+        return SpeakersCalls.query.filter_by(
             event_id=event_id
         ).filter(
-            CallForPaper.start_date <= datetime_now
+            SpeakersCalls.start_date <= datetime_now
```

Comment:
I think wherever there is a "CallForPaper", it is better to keep it singular, SpeakersCall.